config/initializers/zeitwerk.rb

```diff
@@ -0,0 +1,39 @@
+Rails.autoloaders.main.inflector.inflect(
+  'ui' => 'UI',
+  'proxy_api' => 'ProxyAPI',
+  'sti' => 'STI',
+  'dhcp' => 'DHCP',
+  'dns' => 'DNS',
+  'tftp' => 'TFTP',
+  'external_ipam' => 'ExternalIPAM',
+  'bmc' => 'BMC',
+  'ui_notifications' => 'UINotifications',
+  'ipam' => 'IPAM',
+  'ssh' => 'SSH',
+  'ssh_provision' => 'SSHProvision',
+  'ssh_execution_provider' => 'SSHExecutionProvider',
+  'keep_current_request_id' => 'KeepCurrentRequestID',
+  'ec2' => 'EC2',
+  'aws' => 'AWS',
+  'gce' => 'GCE',
+  'aix' => 'AIX',
+  'nxos' => 'NXOS',
+  'vrp' => 'VRP',
+  'sso' => 'SSO',
+  'puppet_ca_certificate' => 'PuppetCACertificate',
+  'url_resolver' => 'URLResolver',
+  'ztp_record' => 'ZTPRecord',
+  'aaaa_record' => 'AAAARecord',
+  'ptr4_record' => 'PTR4Record',
+  'ptr6_record' => 'PTR6Record'
```

Comment:
This is basically workaround for `/lib` directory. Also, could please someone tell me why do we even have `/lib` in `eager_load_paths`? And also we have there code which is tightly bound to the whole application and per my thinking and Rails suggestions we should move it to `app/lib` directory or somehow refactor.

Reply:
We (mainly @stejskalleos) has started to work on exactly this and I believe `/lib` is not autoloaded anymore, we were not sure where everywhere we would need to add require statements, but ideal state is `/lib` content being explicitly required where needed (or from `application.rb`) ... and indeed if there are some strictly application related files that belongs to lib, those should be in `app/lib`